From 1878b9d1fd0f493c05122565b569fc079dc433cf Mon Sep 17 00:00:00 2001
From: Jeffrey Clark <h0tw1r3@users.noreply.github.com>
Date: Sat, 3 Aug 2024 19:25:53 +0000
Subject: [PATCH 1/4] (ospuppet) fix virt-what source

currently people.redhat.com does not include the intermediate cert
---
 configs/components/virt-what.rb | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configs/components/virt-what.rb b/configs/components/virt-what.rb
index e5e17bf..7b836c6 100644
--- a/configs/components/virt-what.rb
+++ b/configs/components/virt-what.rb
@@ -1,8 +1,9 @@
 component "virt-what" do |pkg, settings, platform|
+  checksum = "0147b4b44ae0ee685977aa34dfa9bf30ae8e0eb31b7a6d5c0097d16f830fa6fb6afd7156964fc79f3fd5e82b2f68d921fd5306245cc63a2140f6dddc7fdd0e98"
   pkg.version "1.25"
-  pkg.md5sum "2345f1ec5fa0836bff4071659730ac8f"
+  pkg.sha512sum checksum
 
-  pkg.url "https://people.redhat.com/~rjones/virt-what/files/virt-what-#{pkg.get_version}.tar.gz"
+  pkg.url "https://src.fedoraproject.org/repo/pkgs/virt-what/virt-what-#{pkg.get_version}.tar.gz/sha512/#{checksum}/virt-what-1.25.tar.gz"
   pkg.mirror "#{settings[:buildsources_url]}/virt-what-#{pkg.get_version}.tar.gz"
 
   pkg.replaces 'pe-virt-what'
-- 
2.25.1

